### PR TITLE
Warehouse3

### PR DESCRIPTION
**Pull request by TumiMasilo**

### CODE REVIEW

**Review by NoniiM — APPROVED** [review]
Awe!!

**Review by ThapeloNWU — APPROVED** [review]
hectic

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -38,7 +38,14 @@
             android:name=".StockActivity"
             android:parentActivityName=".WarehouseActivity" />
 
-        <activity android:name=".MainActivity"></activity>
```
Comment (TumiMasilo) [inline app/src/main/AndroidManifest.xml]: removed

File: app/src/main/java/com/drms/doneretail/CompletedOrders.java [hunk 2]
```diff
@@ -20,12 +22,17 @@ public CompletedOrders() {
         // Required empty public constructor
     }
 
+    private static String[] CompletedOrders = {"No orders completed yet"};
 
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
         // Inflate the layout for this fragment
-        return inflater.inflate(R.layout.fragment_completed_orders, container, false);
```
Comment (TumiMasilo) [inline app/src/main/java/com/drms/doneretail/CompletedOrders.java:28]: removed

File: app/src/main/java/com/drms/doneretail/PageAdapter.java [hunk 3]
```diff
@@ -24,10 +24,8 @@ public Fragment getItem(int position) {
             case 1:
                 return new StockOnHand();
             case 2:
-                return new ReturnedStock();
```
Comment (TumiMasilo) [inline app/src/main/java/com/drms/doneretail/PageAdapter.java:27]: removed

**Review by TumiMasilo — COMMENTED** [review]
changes

**Pull request merged**